docs/proposals/images/machine-states-preboot/Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
```suggestion
# Copyright 2019 The Kubernetes Authors.
```

Reply:
Using last year's copyright is my way of making this work appear more mature and stable :laughing: